dim.sh

```diff
@@ -20,6 +20,11 @@ cd $FILE_NAME
 tr -d '\015' < setup.sh > setup2.sh
 mv setup2.sh setup.sh
 
+# DIM installation needs `gmake` (`make` is not sufficient)
```

Comment:
Shouldn't we patch DIM and propose the patch upstream?

Reply:
We could contact Clara Gaspar indeed. The project is in maintenance but they take patches.

Reply:
That was the original proposal discussed with @matthiasrichter, but we don't know the reaction time and it breaks the recipe for old releases (given that the oldest release probably dates back to years ago...) I actually propose we do both.

Reply:
I think that it makes sense to do both